gammapy/morphology/models.py

```diff
@@ -67,6 +67,29 @@ class Shell2D(Fittable2DModel):
             \\right.
 
     With :math:`r_{out} = r_{in} + \\mathrm{width}`.
+
+
+    Examples
+    --------
+    .. plot::
+        :include-source:
+
+        import numpy as np
+        from gammapy.morphology import Shell2D
+        import matplotlib.pyplot as plt
+        
+        x0, y0 = 25, 25
```

Comment:
Directly pass those parameters to the constructor?
Saves two lines and the code is just as readable IMO.